### PR TITLE
Product

### PR DESCRIPTION
**Pull request by MigleMic**

### CODE REVIEW

**Comment by martynasslav** [comment]
Repositories should be inside the folder 'Repositories', not 'Repository'.

File: PoSSapi/PoSSapi.csproj [hunk 1]
```diff
@@ -10,6 +10,10 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.EntityFrameworkCore.Sqlite" Version="7.0.1" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Tools" Version="7.0.1">
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+      <PrivateAssets>all</PrivateAssets>
+    </PackageReference>
```
Comment (martynasslav) [inline PoSSapi/PoSSapi.csproj]: We don't need this package any more.

**Review by martynasslav — APPROVED** [review]
LGTM :)

**Review by karoluzas — APPROVED** [review]
Looks great!

**Pull request merged**